Add serialization support for dynamic textures

kcoley opened the pull request:
Fixes https://github.com/BabylonJS/Babylon.js/issues/4793

deltakosh commented on src/Materials/Textures/babylon.dynamicTexture.ts:
```diff
@@ -178,6 +178,20 @@
+         * @returns a serialized dynamic texture object
+         */
+        public serialize(): any {
+            const serializationObject = super.serialize();
```
Does it work on IE?

kcoley replied:
No, this does not work in IE.  I'll work on a fix.

deltakosh replied:
A fix or if none found a check to make sure it won't crash

kcoley commented:
@deltakosh `toDataURL` works on IE11, but my playground was using a promise, which didn't work.  I changed it to use observable instead to test on IE11: http://playground.babylonjs.com/#FU0ES5

I also added a warning indicating that the scene should be ready before serializing the dynamic texture.  I can add this warning to the Dynamic Texture documentation as well.

deltakosh commented:
Good!
